hw2/05-form.js

```diff
@@ -1 +1,50 @@
 // Add your code here
+const form = document.getElementById("form");
+
+document.getElementById("resetBtn").addEventListener("click", clearForm);
+let userName = document.getElementById("userName");
+let userEmail = document.getElementById("userEmail");
+let regStatus = document.getElementById("regStatus");
+let progLangChecked = document.getElementById("progLang");
+let opSysChecked = document.getElementById("opSys");
+let fsWebDevChecked = document.getElementById("fsWebDev");
+let otherInfo = document.getElementById("otherInfo");
+
+function logSubmit() {
+  console.log(userName.value);
+  console.log(userEmail.value);
+  console.log(regStatus.value);
+  console.log(progLangChecked.checked);
+  console.log(opSysChecked.checked);
+  console.log(fsWebDevChecked.checked);
+  console.log(otherInfo.value);
+}
+function clearForm() {
```

Comment:
I found out when I was working on the hw that when you implement a `<button>` and assign `type="reset"`, the form will clear everything automatically.  I guess this works too.